CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+- [Why GitHub](#why-github)
+- [Issues](#issues)
+- [Pull request](#pull-requests-pr)
+- [Participate in our activities](#particpate-in-our-activities)
+- [Code of conduct](#code-of-conduct)
+- [Contact us](#contact-us)
+- [Acknowledgements](#acknowledgements)
+
+## Why contribute
+
+**A key mission of TRIC-DT is to [democratise access to open, reproducible and trustworthy ecosystems of digital twins](https://www.turing.ac.uk/research/research-projects/tric-dt).**
+
+**This is best acheived through broad and open collaboration in the materials we create and share.**
+
+**We invite you to contribute to these materials and our activites in any way you can, to make sure your perspecties and expertise are included, supporting the development of robust, accessible and equitable outputs.**
+
+## Why GitHub
+
+These materials are being developed in GitHub to ensure atribution and version control. Take a look at the [Github for Collaborative Documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) for a quick overview on why we recomend this processs, and [The Turing Way](https://the-turing-way.netlify.app/index.html) for a more in-depth discussion.
+
+### Github account
+Before you can contribute formally (i.e. add to these materials directly), you'll need a [GitHub account (free)](https://github.com). 
+
+### GitHub contribution processes
+
+You can contribute by [getting in touch with us](#contact-us) and we will record your engagement using  [all-contributors](https://allcontributors.org).
+
+You can also contribute directly to these materials GitHub processes ([issues](#issues) and [pull requests](https://cassgvp.github.io/github-for-collaborative-documentation/docs/tut/3-2-fork-their-repo.html))
+
+## Issues
+Github issues are a way of publicly recording conversations relating to decision making, and the creation of outputs. They can also be used in a project team to stage and prioritise activity and delivery of materials. See [The Turing Way](https://the-turing-way.netlify.app/communication/os-comms/os-comms-issue-tracking.html) for an in-depth backgounrd on issues and project boards.
+
+*Take a look at our [issues](https://github.com/alan-turing-institute/tric-dt/issues) to see what we're working on, and our [project board](https://github.com/orgs/alan-turing-institute/projects/146/views/4) to see how we are planning delivery.*
+
+### Commenting on issues
+Issues are actionable conversations with atribution and emojis :grin:. We uses issues to record decision making, and we invite you to contribute to that process by commenting on an issue! 
+
+**Take a look at this [issue about our README](https://github.com/alan-turing-institute/tric-dt/issues/17) for an example of how we work with issues :rocket:**
+
+### Open a new issue
+If you'd like to start a conversation about something which isn't already an open issue, you'd love you to [open a new one](https://github.com/alan-turing-institute/tric-dt/issues/new/choose)! 
```

Comment:
Spelling:
- you'd love you to -> we'd love you to

Reply:
maybe also add something about that this link takes you to not just "create a new issue" but the template selector?